.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - 4
```

Comment:
👍 We're dropping this soon for Koa 2 support anyway.

Reply:
Cool, then.